timemachine/fe/atom_mapping.py

```diff
@@ -187,11 +193,15 @@ def get_core_by_mcs(mol_a, mol_b, query, threshold=0.5):
     matches_a = mol_a.GetSubstructMatches(query, uniquify=False)
     matches_b = mol_b.GetSubstructMatches(query, uniquify=False)
 
-    # cost[i, j] = sum_i distance(conf)
     cost = np.zeros((len(matches_a), len(matches_b)))
     for i, a in enumerate(matches_a):
```

Comment:
I noticed that tests still pass when `matches_a` is substituted with `matches_b` on this line. Is it easy to add an example that would fail?

Reply:
I'm not sure what the old tests are doing - I think we're going to 1) remove the old tests and the old atom_mapping code and 2) rewrite them based on this class (and whatever protocols we support).